src/Arcus.Templates.ServiceBus.Topic/Program.cs

```diff
@@ -61,14 +61,20 @@ public static IHostBuilder CreateHostBuilder(string[] args)
                            configuration.AddCommandLine(args);
                            configuration.AddEnvironmentVariables();
                        })
+                       .ConfigureSecretStore((config, stores) =>
+                       {
+//[#if DEBUG]
+                           stores.AddConfiguration(config);
+//[#endif]
+
+                           //#error Please provide a valid secret provider, for example Azure Key Vault: https: //security.arcus-azure.net/features/secrets/consume-from-key-vault
+                           // stores.AddAzureKeyVault(secretProvider: null);
```

Comment:
I would uncomment this and use the overload which requires the vault name and use MSI by default

Reply:
I added the NuGet package that contains this extension and left it there without any conditions in the csproj file. Therefore, it will be included by default (even without any authentication options).